core/src/main/java/ai/timefold/solver/core/impl/heuristic/selector/move/generic/RuinRecreateConstructionHeuristicPhaseBuilder.java

```diff
@@ -58,6 +58,7 @@ public EntityPlacer<Solution_> getEntityPlacer() {
 
     @Override
     public DefaultConstructionHeuristicPhase<Solution_> build() {
+        this.setEnableCollectMetrics(false);
```

Comment:
Arguably, if this is only used in one place and the default is true, this should rather be called `disableMetricCollection()`, inverting the logic.